code/script.js

```diff
@@ -222,44 +226,57 @@ const generateBoard = () => {
   })
 }
 
+
 // Randomly select a person from the characters array and set as the value of the variable called secret
 const setSecret = () => {
   secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
 }
 
 // This function to start (and restart) the game
 const start = () => {
-  // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
+  generateBoard()
+  setSecret()
+  selectQuestion()
```

Comment:
I put it here because otherwise the "Find out"-button doesn't work from the start. You actually have to choose something else from the list before it's activated. But if you want to ask the first thing on the list that is set from the start of the game this made the button work. But there might be a smarter way to solve it.